.factory/library/architecture.md

```
@@ -0,0 +1,119 @@
+
+High-level system understanding: components, relationships, data flows, invariants. NOT implementation details.
+
+---
+
+## System Overview
+
+Comparative Rideshares is a Next.js 14 App Router application that compares prices across Uber, Lyft, Taxi, and Waymo in the Bay Area. It uses algorithmic pricing (no real rideshare API integration).
+
+## Core Data Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)